resources/js/components/home-footer.js

```diff
@@ -52,32 +61,95 @@ class HomeFooter extends LitElement {
         --mod-actionbutton-content-color-default: #ffff !important;
         --mod-actionbutton-height: 40px;
         --mod-actionbutton-edge-to-text: 20px;
-        --mod-popover-border-color: #f8f6f6;
+        --mod-popover-border-color: rgb(66, 64, 64);
         --mod-popover-corner-radius: 5px;
       }
 
       @media (hover: hover) {
         :host(:hover) {
           --mod-actionbutton-background-color-hover: #3F729B;
           --mod-actionbutton-content-color-hover: #ffff;
-          --mod-popover-border-color: #f8f6f6;
+          --mod-popover-border-color: rgb(66, 64, 64);
         }
       }
     `;
   }
 
-  render() {
-    const currentUrl = window.location.href; // Gets the current URL
 
+  connectedCallback() {
```

Comment:
right click function name -> AI Actions > Write Documentation for each class and function/method in this pull request. After AI generates the DocBlock, edit to make sure the documentation it adds makes sense. 
![Screenshot from 2024-02-23 11-08-04](https://github.com/thecodezone/dt-home/assets/5910297/ccb27543-23f7-4fe0-b6f3-3a26d3f9face)